test/TestExpectations.json

```diff
@@ -2198,7 +2200,7 @@
     "testIdPattern": "[frame.spec] Frame specs Frame Management should detach child frames on navigation",
     "platforms": ["darwin", "linux", "win32"],
     "parameters": ["firefox", "webDriverBiDi"],
-    "expectations": ["PASS"]
+    "expectations": ["FAIL"]
```

Comment:
What caused a change for this expectation?

Reply:
Added comment.